zokrates_stdlib/stdlib/utils/pack/bool/nonStrictUnpack256.zok

```diff
@@ -1,11 +1,11 @@
-#pragma curve bn128
-
 import "./unpack_unchecked";
 
 // Unpack a field element as 256 big-endian bits
 // Note: uniqueness of the output is not guaranteed
 // For example, `0` can map to `[0, 0, ..., 0]` or to `bits(p)`
-def main(field i) -> bool[256] {
-    bool[254] b = unpack_unchecked(i);
-    return [false, false, ...b];
+def main(field i, u32 bit_size) -> bool[256] {
+    assert(bit_size == 254 || bit_size == 255);
```

Comment:
Is there a reason why this two bit sizes are only allowed here? Maybe `assert(bit_size <= 256);` would fit better?

Reply:
Good point. I found out that for the baby jubjub and jubjub the field bit size if 254 and 255 respectively ([resource](https://neuromancer.sk/std/other/JubJub)). So I went for a strict assertion as the nonStrictUnpack is only used for signatures in the code.